src/daemon/daemon.cpp

```diff
@@ -114,21 +114,38 @@ mp::Query query_from(const mp::LaunchRequest* request, const std::string& name)
     return {name, image, false, request->remote_name(), query_type, true};
 }
 
-auto make_cloud_init_vendor_config(const mp::SSHKeyProvider& key_provider, const std::string& time_zone,
-                                   const std::string& username, const std::string& backend_version_string)
+auto make_cloud_init_vendor_config(const mp::SSHKeyProvider& key_provider, const std::string& username,
+                                   const std::string& backend_version_string, const mp::CreateRequest* request)
 {
     auto ssh_key_line = fmt::format("ssh-rsa {} {}@localhost", key_provider.public_key_as_base64(), username);
+    QString pollinate_alias = QString::fromStdString(request->image());
 
+    if (pollinate_alias.isEmpty())
+    {
+        pollinate_alias = "default";
+    }
+    else if (pollinate_alias.startsWith("http"))
```

Comment:
not to be too pedantic, but shouldn't this check for `http://` and `https://`?

Reply:
Actually, upon further consideration, I think this should stay due to this already occurring in the following lines:
https://github.com/canonical/multipass/blob/release/1.11/src/daemon/daemon.cpp#L109-L112

Reply:
okay, makes sense!